#! /bin/sh /usr/share/dpatch/dpatch-run
## dbug433884.dpatch by  Faidon Liambotis <paravoid@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: http://bugs.debian.org/433884

@DPATCH@
diff -urNad asterisk-1.4.8~dfsg~/main/Makefile asterisk-1.4.8~dfsg/main/Makefile
--- asterisk-1.4.8~dfsg~/main/Makefile	2007-07-03 21:17:31.000000000 +0100
+++ asterisk-1.4.8~dfsg/main/Makefile	2007-07-21 10:20:51.000000000 +0100
@@ -120,13 +120,6 @@
 AST_EMBED_LIBS:=$(foreach dep,$(EMBED_LIBS),$(value $(dep)))
 OBJS:=$(sort $(OBJS))
 
-ifneq ($(wildcard ../channels/h323/Makefile.ast),)
-  include ../channels/h323/Makefile.ast
-else
-  H323LDFLAGS=
-  H323LDLIBS=
-endif
-
 asterisk: $(OBJS) editline/libedit.a db1-ast/libdb1.a $(AST_EMBED_LDSCRIPTS)
 	@$(ASTTOPDIR)/build_tools/make_build_h > $(ASTTOPDIR)/include/asterisk/build.h.tmp
 	@if cmp -s $(ASTTOPDIR)/include/asterisk/build.h.tmp $(ASTTOPDIR)/include/asterisk/build.h ; then echo ; else \
@@ -135,7 +128,7 @@
 	@rm -f $(ASTTOPDIR)/include/asterisk/build.h.tmp
 	@$(CC) -c -o buildinfo.o $(ASTCFLAGS) buildinfo.c
 	$(ECHO_PREFIX) echo "   [LD] $^ -> $@"
-	$(CMD_PREFIX) $(CXX) $(STATIC_BUILD) -o $@ $(ASTLINK) $(AST_EMBED_LDFLAGS) $(ASTLDFLAGS) $(H323LDFLAGS) $^ buildinfo.o $(AST_LIBS) $(AST_EMBED_LIBS) $(H323LDLIBS)
+	$(CMD_PREFIX) $(CC) $(STATIC_BUILD) -o $@ $(ASTLINK) $(AST_EMBED_LDFLAGS) $(ASTLDFLAGS) $^ buildinfo.o $(AST_LIBS) $(AST_EMBED_LIBS)
 	@$(ASTTOPDIR)/build_tools/strip_nonapi $@
 
 clean::
